examples/typescript/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
@valentyn88 are we taking this example from cypress examples? can we link back to their documentation?

Reply:
Yes, this example was taken from cypress. It's a JSON file and I can't add a link to the original file. We also have some other examples from their website or github repos. Maybe we can mention them in our README.md file? Do we need to have a link in they have MIT license?

Reply:
still a good practice in open-source to name the upstream project were talking the examples

Reply:
@vrunoa I added a note about the original repo and added typescript to the workflow.